internal/server/models/group.go

```diff
@@ -12,8 +12,6 @@ type Group struct {
 	CreatedBy         uid.ID
 	CreatedByProvider uid.ID
 
-	Identities []Identity `gorm:"many2many:identities_groups"`
```

Comment:
This still feels like a relation we will want to know (same for identities in an org), whats the reason for removing this?

Reply:
I was struggling to get the initial schema to match the migrated schema. I was trying lots of things, and I don't remember now if this is necessary or just one of the things I tried.

It should be safe to remove this column, it's not used. Pretty soon we'll be writing regular sql that won't need these fields unless we plan on populating them in a query response.

Reply:
I added this back and the test still passes. I'll restore this line for now and we can look to remove it in the future.